File: youtube/views/youtube-views.php
<?php
declare(strict_types = 1);
require 'cove/youtube.php';

if ($argc != 2) {
   echo "youtube-views.php <URL>\n";
   exit(1);
}

$url_s = $argv[1];
$query_s = parse_url($url_s, PHP_URL_QUERY);
parse_str($query_s, $query_m);
$id_s = $query_m['v'];
$info_o = youtube_info($id_s);
echo youtube_views($info_o), "\n";

File: youtube/views/youtube-views.go
package main

import (
   "log"
   "musicdb/youtube"
   "net/url"
   "os"
)

func main() {
   if len(os.Args) != 2 {
      println("youtube-views <URL>")
      os.Exit(1)
   }
   url_s := os.Args[1]
   o, e := url.Parse(url_s)
   if e != nil {
      log.Fatal(e)
   }
   id_s := o.Query().Get("v")
   m := youtube.Info(id_s)
   view_s := youtube.Views(m)
   println(view_s)
}
